build/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.19-alpine as builder
+
+ENV GO111MODULE=on
+ENV CGO_ENABLED=0
+WORKDIR /work
+
+ADD go.mod go.sum /work/
+RUN go mod download
+
+ADD . .
+RUN go build -o build/kubevuln cmd/http/main.go
+
+FROM alpine
+
+RUN addgroup -S ks && adduser -S ks -G ks
+USER ks
+WORKDIR /home/ks/
```

Comment:
suggestion (non-blocking): since `ks` gets repeated multiple times, should we extract it into an `ARG`?

Could help re-use the value in the future.

Reply:
this is copied from operator... if it was me I would run a randomUID and use `scratch` as the base image